### ISSUE TITLE
How not to miss tasks requiring my contribution

### ISSUE THREAD

**Issue by olivierlemee** [cybnity/mpp-demonstrators#18]
screen allowing the test: 
- https://www.notion.so/cybnity/Recipient-s-attention-of-new-assigned-tasks-is-visually-captured-to-animate-his-motivation-to-treat--9aabdef3f9c94acdbc09cc0b6aa38ef6
- https://www.notion.so/cybnity/The-treatment-of-an-urgent-high-level-task-is-quickly-started-by-its-assigned-actor-who-was-not-conn-660e38282fbd4d738e003473ef0ffa7e

**Comment by olivierlemee** [comment]
ok pour ces deux endroits de capture d'attention sur une action prioritaire devant capter son attention. Par contre, réaligner les textes qui ne sont pas correctes en terme de libellé de rubrique avant de le passer "terminé"

**Comment by cosmyx82** [comment]
Oui, je passerai l'ensemble de la tache qd les 2 tickets seront valider

**Comment by olivierlemee** [comment]
- indicateur d'alertcon: commentaire inséré car on ne voit pas les index de chaque niveau et on ne voit pas la situation actuelle
- panneau react : même commentaire à intégrer que déjà formulées dans le scénario " a task defined..." du dessous
- panneau gérer: l'aperçu et l'indication de la priorité n'est pas identiquement traitée que sur le scénario "a task is defined...". Aligner les deux pour qu'une seule représentation doit validée

**Comment by olivierlemee** [comment]
l'écran wireframé ne supporte que le test "recipient attention..." mais n'oublies par qu'il y a un scénario entrant qui différe sur le test "the treatment of an urgent..." (https://www.notion.so/cybnity/The-treatment-of-an-urgent-high-level-task-is-quickly-started-by-its-assigned-actor-who-was-not-conn-660e38282fbd4d738e003473ef0ffa7e) qui amène à devoir montrer l'atterrissage dans l'ui puisque dans la description du scénario, tu peux lire qu'il y a des contraintes fonctionnelles complémentaires concernant l'acceptation de prise en charge d'une action soumise à l'utilisateur lorsqu'il peux la déléguer ou l'accepter

**Comment by olivierlemee** [comment]
@cosmyx82 pense toujours à bien regarder le description de la tâche github lorsque tu la traites, car parfois dans la même tâche je t'ai regroupé plus de 1 scénario de test à supporter lorsqu'ils s'avèrent très proche ou sur même sujets. Et donc que tu n'oublies pas de traiter les deux dans ton wireframe, que ce soit avec un seul diagramme ou plusieurs puisque la revue du ticket implique les deux comme ici d'après la description initiale "screen allowing test"

**Comment by cosmyx82** [comment]
J'ai crée des templates, on gagnera du temps sur les modifications récurrentes

**Comment by olivierlemee** [comment]
en me le réaffectant, peux-tu préciser lequel des 2 usermap je dois revoir cette fois (corrections appliquées sur le premier déjà revue ou sur l'autre)?

**Comment by olivierlemee** [comment]
revue scénario "recipient..."
- panneau "gérer"
   - le texte d'invitation dans la zone de recherche "search" a été remplacé par sa version traduite en "rechercher"
   - au-dessus ou à côté du sélecteur de périmètre de recherche, il n'existe aucun libellé permettant de comprendre à quoi sert ce sélecteur. Ajouter une information pour que l'utilisateur sans action de sa part, comprenne et "voit" à quoi sert ce sélecteur: commentaire précis ajouté sur le sélecteur au sein du wireframe
   - étant dans un scénario où l'écran montre une partie droite vide, montrer que le layout d'items prends l'ensemble de l'espace et ne reste pas juste sur les 2/3 de l'écran en horizontal (on voit le comportement différent lorsque le panneau "react" est affiché au sein d'autres scénario; donc ici montrer le cas où "gérer" profite de tout l'écran.
 VALIDATION : @cosmyx82  une fois ces corrections appliquées, je valide que ce scénario est considéré conforme et tu peux le passer en état suivant "ready for uat".

comme dans ce ticket il y avait initialement le 2ème scénario passé en low concernant "https://www.notion.so/cybnity/The-treatment-of-an-urgent-high-level-task-is-quickly-started-by-its-assigned-actor-who-was-not-conn-660e38282fbd4d738e003473ef0ffa7e". , je mets ce scénario dans un ticket indépendant remis au backlog. permettant ainsi que ce ticket reste uniquement dédié au scénario de test "recipient..."

**Comment by olivierlemee** [comment]
@cosmyx82  revue scénario "recipient":
- la visual alert (cloche) sur le panneau principal en haut à droit semble non activée, pour répondre au scénario qui dit "qu'on capture l'attention de l'utilisateur que des actions urgentes sont à traiter (objectif de l'écran : attirer son attention). Peut-être un bug de ton composant visuel de cloche
Hormis cela que je te remercie d'actualiser, on peux considérer ce scénario validé pour la la suite

**Comment by cosmyx82** [comment]
Tu peux faire la review de ces 2 scenarios

**Comment by olivierlemee** [comment]
statu revue: ok validé

**Comment by cosmyx82** [comment]
https://www.figma.com/file/xWxHVvn0Vools7ETTiMVWa/Wire-Frame?type=design&node-id=2554%3A156271&mode=design&t=zLdCV7CW1esg6mBJ-1

**Comment by olivierlemee** [comment]
le cas est ok mais il existe des commentaires à traiter pour alignement des corrections souhaitées sur identiques composants

**Comment by olivierlemee** [comment]
ok validé
